src/components/two-factor-authentication/SelfServiceWorkflow.vue

```diff
@@ -151,35 +151,47 @@
             </v-row>
           </v-container>
           <v-container v-if="workflowStep === TwoFactorSteps.QRCode">
-            <v-row class="text-body bold px-md-16">
-              <div class="v-col">
-                <p class="text-body">
-                  {{ $t('admin.person.twoFactorAuthentication.pleaseScan') }}
-                </p>
-              </div>
-            </v-row>
-            <v-row
-              v-if="twoFactorStore.qrCode.length === 0"
-              class="justify-center"
-            >
-              <v-progress-circular
-                size="250"
-                width="250"
-                indeterminate
+            <v-container v-if="!twoFactorStore.errorCode">
+              <v-row class="text-body bold px-md-16">
+                <div class="v-col">
+                  <p class="text-body">
+                    {{ $t('admin.person.twoFactorAuthentication.pleaseScan') }}
+                  </p>
+                </div>
+              </v-row>
+              <v-row
+                v-if="twoFactorStore.qrCode.length === 0"
+                class="justify-center"
               >
-              </v-progress-circular>
-            </v-row>
-            <v-row
-              v-if="twoFactorStore.qrCode.length > 0"
-              class="justify-center"
-            >
-              <v-img
-                class="printableContent image-width"
-                :src="twoFactorStore.qrCode"
-                max-width="250"
-                data-testid="software-token-dialog-qr-code"
-              />
-            </v-row>
+                <v-progress-circular
+                  size="250"
+                  width="250"
+                  indeterminate
+                >
+                </v-progress-circular>
+              </v-row>
+              <v-row
+                v-if="twoFactorStore.qrCode.length > 0"
+                class="justify-center"
+              >
+                <v-img
+                  class="printableContent image-width"
```

Comment:
```suggestion
                  class="printable-content image-width"
```
Klassen bitte in kebab-case, damit es überall einheitlich ist.